gostream/shell.nix

```diff
@@ -1,14 +0,0 @@
-{ pkgs ? import (fetchTarball
```

Comment:
I keep seeing shell.nix

Was Nix OS support ever important for Viam?

Reply:
I think the repo was straight up copy/pasted into rdk https://github.com/edaniels/gostream